README.md

```diff
@@ -109,6 +112,85 @@ my-project/themes/apac (mapped to src/entries/themes/apac.ts)
 If a dependency is present in `devDependencies` (but not in `peerDependencies`) it is bundled along with the project's source code.
 `dependencies`, `peerDependencies` and `optionalDependencies` are marked as external and not bundled.
 
+#### Package mode
+
+```bash
+crackle package --mode=bundle
+crackle package --mode=preserve
+```
+
+This controls how Crackle generates output files.
+
+- `bundle` rolls up output files into as few chunks as possible (default behaviour)
+- `preserve` creates separate files for all modules using the original module names as file names.
+  This is similar to Rollup's [`output.preserveModules`](https://rollupjs.org/configuration-options/#output-preservemodules), but allows more fine-grained control because we hook into [`output.manualChunks`](https://rollupjs.org/configuration-options/#output-manualchunks).
+
+The mode can also be configured via `crackle.config.ts`:
+
+```ts
+// crackle.config.ts
+import { defineConfig } from '@crackle/cli/config';
+
+export default defineConfig({
+  package: {
+    mode: 'preserve',
+  },
+});
+```
+
+#### DTS mode
+
+Some libraries declare namespaces, which are hard/impossible to bundle.
+For such cases, Crackle has an option to preserve the file structure of the generated `.d.ts` files.
+
+```ts
+// crackle.config.ts
+import { defineConfig } from '@crackle/cli/config';
+
+export default defineConfig({
+  dts: {
+    mode: 'preserve',
+  },
+});
+```
+
+- `bundle` rolls up output files into as few chunks as possible (default behaviour)
+- `preserve` creates separate files for all modules using the original module names as file names.
+  This is similar to Rollup's [`output.preserveModules`](https://rollupjs.org/configuration-options/#output-preservemodules).
+
+### `crackle dev`
+
+Generate entry points for local development.
+This will generate stub entry points for local development.
+Stub entry points import the source files directly instead of the compiled files.
+
+#### Shim mode
+
+By default, Crackle generates a `require` shim which enables the loading of TypeScript files in Node.js using a require hook.
+There are situations where this is not _required_, such as when using a bundler that supports TypeScript natively, e.g. esbuild or Vite.
```

Comment:
Feel like "not" should be the emphasis here. Thoughts?
```suggestion
There are situations where this is _not required_, such as when using a bundler that supports TypeScript natively, e.g. esbuild or Vite.
```

Reply:
I was making a pun; didn't want to draw _too_ much attention to it, but still a little.